examples/reference/elements/matplotlib/Image.ipynb

```diff
@@ -82,10 +82,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Points (color='black' marker='x' size=20)\n",
     "closest = img.closest((0.1,0.1))\n",
+    "points = hv.Points([closest])\n",
+    "\n",
     "print('The value at position %s is %s' % (closest, img[0.1, 0.1]))\n",
-    "img * hv.Points([img.closest((0.1,0.1))])"
+    "\n",
+    "img * points.opts(color='black', marker='x', s=60)"
```

Comment:
I would leave this, but following the rules this should not be inlined.

Reply:
i.e. it would be:

```python
points = hv.Points([closest]).opts(color='black', marker='x', s=60)
img * points
```

Reply:
I would do:

```python
(img * points).opts(
   opts.Points(color='black', marker='x', s=60))
```

Again for consistency.

Reply:
Since we're only styling one of the elements I find my version more readable.